File: FrontEnd/src/modules/Trader/models/SymbolInfoView.ts
export interface SymbolInfoView {
  id: string;
  symbolName: string;
  symbolIsin: string;
  price: number;
  quantity: number;
  side: number;
  validityType: number;
  commission: number;
  orderModelType: number;
  orderFrom: number;
}

export interface MarketSymbolInfoView {
  symbolIsin: string;
  highAllowedPrice?: number | null;
  lowAllowedPrice?: number | null;
  lastTradedPrice?: number | null;
  closingPrice?: number | null;
  tradeDate?: string | null;
  fetchedAt?: number;
}

File: Trader.Application/Dtos/SymbolInfoView.cs
namespace Trader.Application.Dtos
{
    public class SymbolInfoView
    {
        public Guid Id { get; set; }
        public string SymbolName { get; set; } = default!;
        public string SymbolIsin { get; set; } = default!;
        public long Price { get; set; }
        public long Quantity { get; set; }
        public int Side { get; set; }
        public int ValidityType { get; set; }
        public decimal Commission { get; set; }
        public int OrderModelType { get; set; }
        public int OrderFrom { get; set; }
    }
}